pkg/scheduler/framework/plugins/default_registry.go

```diff
@@ -31,3 +31,22 @@ func NewDefaultRegistry() framework.Registry {
 		noop.Name: noop.New,
 	}
 }
+
+// Uncomment this function once the first predicate/priority configurator is registered.
+// func appendToPluginSet(pluginSet *config.PluginSet, name string, weight int32) *config.PluginSet {
+// 	if pluginSet == nil {
+// 		pluginSet = &config.PluginSet{}
+// 	}
+// 	pluginSet.Enabled = append(pluginSet.Enabled, config.Plugin{Name: name, Weight: weight})
```

Comment:
It seems no need to add them to enabled plugin sets when registering predicate/priority functions, it just means that admins could use them by configuring.

Reply:
This is not for registering the plugins in the default registry, this is for registering a config producer that creates a configuration for a mapped predicate/priority.